homeassistant/components/enigma2/media_player.py

```diff
@@ -143,7 +147,15 @@ def __init__(self, name: str, device: OpenWebIfDevice, about: dict) -> None:
 
     async def async_turn_off(self) -> None:
         """Turn off media player."""
-        await self._device.turn_off()
+        try:
+            await self._device.turn_off()
+        except ServerDisconnectedError as err:
+            _LOGGER.warning(
```

Comment:
Nowadays we raise `HomeAssistantError` from service handlers on error.

https://developers.home-assistant.io/docs/core/platform/raising_exceptions/

Reply:
But this case is a expected error. "Deep standby" is the enigma2 term for shutdown, so it is expected that the device is not reachable anymore. But when I raise `HomeAssistantError`, then it is handled as if the service call "turn_off" failed.

Reply:
The service call does fail, right?

Reply:
You mean it turns off and the error is expected? Then why would we log a warning?

Reply:
The service call is successful (the device turns off), but the connection is not closed gracefully by the device before shutting down, so causing the "ServerDisconnectedError" by aiohttp.

Reply:
> You mean it turns off and the error is expected? Then why would we log a warning?

Good point, I will remove the warning.